src/poetry/installation/executor.py

```diff
@@ -881,7 +881,7 @@ def _download_archive(self, operation: Install | Update, link: Link) -> Path:
                 return archive
             except:
                 # If we failed to download with curl, give it another try with the local implementation
-                logger.warn('Failed to download archive with curl. trying again')
+                logger.info('Failed to download archive with curl. trying again')
```

Comment:
```suggestion
                logger.warn('Failed to download archive with curl. trying again')
```

Reply:
I don't want warn because want to hide them by default. They will show up with `-v` with info.